artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/openmbean/OpenTypeSupport.java

```diff
@@ -293,7 +293,7 @@ protected void init() throws OpenDataException {
             if (m.containsProperty(Message.HDR_LARGE_COMPRESSED)) {
                rc.put(CompositeDataConstants.TEXT_BODY, "[compressed]");
             } else {
-               final String text = m.getReadOnlyBodyBuffer().readString();
+               final String text = m.getReadOnlyBodyBuffer().readNullableSimpleString().toString();
```

Comment:
This shouldn't be
```java
final SimpleString nullableText = m.getReadOnlyBodyBuffer().readNullableSimpleString();
fina String text = nullableText == null? null : nullableText.toString();
// ...
```
because of
```java
   public static SimpleString readNullableSimpleString(ByteBuf buffer) {
      int b = buffer.readByte();
      if (b == DataConstants.NULL) {
         return null;
      }
      return readSimpleString(buffer);
   }
```

Reply:
@franz1981 thanks, just fixed